src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -130,6 +133,28 @@
           <ButtonText>&amp;Package Manager Settings</ButtonText>
         </Strings>
       </Button>
+
+      <Button guid="guidDialogCmdSet" id="cmdidUpdatePackage" priority="0x0100" type="Button">
```

Comment:
The update button shouldn't have any icon for now? forever? @chgill-MSFT wasn't there some arrow indicator we decided on? Or do you want it blank for now?

Reply:
The update menu items should not have an icon for now. I confirmed this with @chgill-MSFT .